packages/wrangler/src/whoami.tsx

```diff
@@ -65,11 +65,11 @@ function Permissions(props: {
 				</Text>
 				<Text>Scope (Access)</Text>
 				{permissions.map(([type, name]) => (
```

Comment:
Probably better to use index as the key, as `name` will be duplicated
Here name actually is the 'read' / 'write' or sometimes not defined (e.g. for offline_access)
I guess we could use `type` as the key but there could be case where it is duplicated with both read and write so probably safest to use the index

Reply:
I'll do `name-index` should prevent any overlap

Reply:
I will admit, when `type` and `name` are not the most descriptive variable names for what they actually are... `scope` and `access` might be better (what we supply as the heading names in the cli output)